generators/package/templates/src/main/assemble/assembly.xml

```diff
@@ -144,6 +144,13 @@
       </filterset>
     </copy>
 
+<% if (v.isAfterOrEquals('9.10-SNAPSHOT')) { -%>
```

Comment:
I would use after `9.10` here because it doesn't make sense to have a package relying on `9.10-SNAPSHOT` these days :)

Reply:
Not even for hotfixes?